### DISCUSSION TITLE
"Read state" is OK, but "Set State" does not work anymore  (Solved)

### DISCUSSION THREAD

**Discussion by ambitzgi** [Koenkk/zigbee2mqtt#19537] [General]
Hi to everyone
Since today I cannot change a state of any device anymore. For example I cannot switch on any light. 
However when I switch the light manually, the change of the state works perfectly. 

Here is some lines from the log file (switching manually on and  off):

info  2023-11-03 09:37:16: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":72,"power_on_behavior_l1":"off","state":"OFF","state_l1":"ON","state_l2":"OFF"}'
debug 2023-11-03 09:37:16: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:16: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":72,"power_on_behavior_l1":"off","state":"OFF","state_l1":"ON","state_l2":"OFF"}'
debug 2023-11-03 09:37:17: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:17: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":69,"power_on_behavior_l1":"off","state":"OFF","state_l1":"ON","state_l2":"OFF"}'
debug 2023-11-03 09:37:17: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:17: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":72,"power_on_behavior_l1":"off","state":"OFF","state_l1":"ON","state_l2":"OFF"}'
debug 2023-11-03 09:37:17: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:17: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":69,"power_on_behavior_l1":"off","state":"OFF","state_l1":"ON","state_l2":"OFF"}'
debug 2023-11-03 09:37:18: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:18: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":72,"power_on_behavior_l1":"off","state":"OFF","state_l1":"OFF","state_l2":"OFF"}'
debug 2023-11-03 09:37:19: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:19: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":69,"power_on_behavior_l1":"off","state":"OFF","state_l1":"OFF","state_l2":"OFF"}'
debug 2023-11-03 09:37:19: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:19: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":69,"power_on_behavior_l1":"off","state":"OFF","state_l1":"OFF","state_l2":"OFF"}'
debug 2023-11-03 09:37:19: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:19: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":72,"power_on_behavior_l1":"off","state":"OFF","state_l1":"OFF","state_l2":"OFF"}'
debug 2023-11-03 09:37:19: Received Zigbee message from 'MS104BZ_1', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info  2023-11-03 09:37:19: MQTT publish: topic 'zigbee2mqtt/MS104BZ_1', payload '{"linkquality":69,"power_on_behavior_l1":"off","state":"OFF","state_l1":"OFF","state_l2":"OFF"}'

Here is some lines from the log file (switching via Openhab -> zigbee2mqtt, errors: 'MAC channel access failure' (225)):

debug 2023-11-03 09:37:32: Received MQTT message on 'zigbee2mqtt/MS104BZ_1/set' with data 'ON'
debug 2023-11-03 09:37:32: Publishing 'set' 'state' to 'MS104BZ_1'
error 2023-11-03 09:37:42: Publish 'set' 'state' to 'MS104BZ_1' failed: 'Error: Command 0x60a423fffeff3995/1 genOnOff.on({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC channel access failure' (225))'
debug 2023-11-03 09:37:42: Error: Command 0x60a423fffeff3995/1 genOnOff.on({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC channel access failure' (225))
    at ZStackAdapter.sendZclFrameToEndpointInternal (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:415:23)
    at Queue.executeNext (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

My Zigbee channel is 11. I have changed the WLAN channel to Wifi-Channel-11, so the should not overlap. 
Unfortunately this did not help.

Any idea?
Greetings from Switzerland!
Thomas

**Comment by tedsears** [comment]
Thomas, my zigbee2mqtt installation also just stopped working entirely in homeassistant after an upgrade to zigbee2mqtt 1.33.2-1.   It sees similar in that I can see mqtt messages being published, but none of the devices were responding, and I couldn't trigger anything to happen from inside homeassistant.  I don't see any "MAC channel access failure" in the logs.   Downgrading to zigbee2mqtt 1.33.1-1 fixed it in my case....

**Comment by ambitzgi** [comment]
I just had a look in my log file. I still use version 1.33.0 (commit #4d53ac4c). 
I will try to update to 1.33..1.

**Comment by ambitzgi** [comment]
I just upgraded to 1.33.2. Unfortunately the problem still exists. 
Maybe I should try to downgrade to 1.33.1. 
What is the git command to downgrade? 
With git pull you get the newest version.

**Comment by ambitzgi** [comment]
I saw in the log that a device had sent series of identical messages. 
I therefore suspected that one device was running "amok" and was filling up the Zigbee radio channel. 
The easiest way to fix this was to switch the electrical power on and off via a fuse. 
Indeed it helped.
